addons/packages/vsphere-cpi/1.22.5/bundle/config/vsphereconf.lib.txt

```diff
@@ -0,0 +1,68 @@
+
+(@ def vsphere_conf(values): -@)
+[Global]
+secret-name = "cloud-provider-vsphere-credentials"
+secret-namespace = "kube-system"
+(@ if values.vsphereCPI.insecureFlag: -@)
+insecure-flag = "1"
+(@ else: -@)
+thumbprint = "(@=values.vsphereCPI.tlsThumbprint @)"
+(@ end -@)
+[VirtualCenter "(@=values.vsphereCPI.server @)"]
+datacenters = "(@=values.vsphereCPI.datacenter @)"
+(@ if values.vsphereCPI.ipFamily: -@)
+ip-family = "(@=values.vsphereCPI.ipFamily @)"
+(@ end -@)
+(@ if values.vsphereCPI.insecureFlag: -@)
+insecure-flag = "1"
+(@ else: -@)
+thumbprint = "(@=values.vsphereCPI.tlsThumbprint @)"
+(@ end -@)
+(@ if values.vsphereCPI.region or values.vsphereCPI.zone: -@)
+[Labels]
+(@ if values.vsphereCPI.region: -@)
+region = "(@=values.vsphereCPI.region @)"
+(@ end -@)
+(@ if values.vsphereCPI.zone: -@)
+zone = "(@=values.vsphereCPI.zone @)"
+(@ end -@)
+(@ end -@)
+(@ if values.vsphereCPI.vmInternalNetwork or values.vsphereCPI.vmExternalNetwork or values.vsphereCPI.vmExcludeInternalNetworkSubnetCidr or values.vsphereCPI.vmExcludeExternalNetworkSubnetCidr: -@)
+[Nodes]
+(@ if values.vsphereCPI.vmInternalNetwork: -@)
+internal-vm-network-name = "(@=values.vsphereCPI.vmInternalNetwork @)"
+(@ end -@)
+(@ if values.vsphereCPI.vmExternalNetwork: -@)
+external-vm-network-name = "(@=values.vsphereCPI.vmExternalNetwork @)"
+(@ end -@)
```

Comment:
We need to add these INI lines for the exclude config here too https://github.com/vmware-tanzu/community-edition/blob/5aaf4df530fce01545f823492e760229bbbbf644/addons/packages/vsphere-cpi/1.23.0-alpha.1/bundle/config/vsphereconf.lib.txt#L38-L43